mrmustard/physics/triples.py

```diff
@@ -447,3 +447,37 @@ def fock_damping_Abc(n_modes: int) -> Union[Matrix, Vector, Scalar]:
     c = 1.0 + 0j
 
     return A, b, c
+
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#  Maps between representations
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+def displacement_gate_s_parametrized_Abc(s: int) -> Union[Matrix, Vector, Scalar]:
+    r"""
+    The ``(A, b, c)`` triple of a single-mode ``s``-parametrized dispalcement gate ``D(\gamma)``.
```

Comment:
Check the docs built by github, but I'm quite sure that you need the backslash to render properly. Also in the returns

```suggestion
    The ``(A, b, c)`` triple of a single-mode ``s``\-parametrized dispalcement gate ``D(\gamma)``.
```